src/date/expand-pattern.js

```diff
@@ -48,25 +171,38 @@ return function( options, cldr ) {
 			if ( !result ) {
 				timeSkeleton = skeleton.split( /[^hHKkmsSAzZOvVXx]/ ).slice( -1 )[ 0 ];
 				dateSkeleton = skeleton.split( /[^GyYuUrQqMLlwWdDFgEec]/ )[ 0 ];
```

Comment:
Let's defer this initialization by moving them both inside the `else` below that will actually use them.